presto-docs/src/main/sphinx/presto_cpp/features.rst

```diff
@@ -153,6 +153,16 @@ is valid.
 If the time period exceeds the parameter value, the request is rejected as
 authentication failure (HTTP 401).
 
+LinuxMemoryChecker
+------------------
+
+Enable the LinuxMemoryChecker by setting ``PRESTO_ENABLE_LINUX_MEMORY_CHECKER=ON`` 
+to ON in presto-native-execution/CMakeLists.txt.
```

Comment:
```suggestion
to ``ON`` in presto-native-execution/CMakeLists.txt.
```
Format suggestion for consistency with your formatting of the same text in properties.rst.